electrumsv/wallet.py

```diff
@@ -3921,10 +4022,27 @@ def start(self, network: Optional[Network]) -> None:
         for account in self.get_accounts():
             account.start(network)
 
+        main_server: Optional[SelectionCandidate] = None
         if self._network is not None:
-            # TODO(1.4.0) Networking. Every wallet has connections to reference servers. Is this
-            #     the right place to connect?
+            server_url = self._storage.get('main_server', "")
```

Comment:
Why is this code below the petty cash server selection, it relates to the same thing?